pgjdbc/src/test/java/org/postgresql/test/jdbc42/UpdateObject310Test.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc42;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.postgresql.test.TestUtil;
+import org.postgresql.util.PSQLException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.TimeZone;
+
+@RunWith(Parameterized.class)
+public class UpdateObject310Test {
+  private static final TimeZone saveTZ = TimeZone.getDefault();
+
+  private Connection con;
+
+  @Parameterized.Parameters
```

Comment:
Personally I prefer importing `org.junit.runners.Parameterized.Parameters` but that's merely cosmetic.